e2e/tests/reset_params_test.go

```diff
@@ -0,0 +1,89 @@
+//go:build e2e
```

Comment:
```suggestion
```

Reply:
This is the opposite of what's wrong. I left out some `e2e` build constraints. :sweat_smile: